web/pdf_presentation_mode.js

```diff
@@ -342,18 +358,86 @@ var PDFPresentationMode = (function PDFPresentationModeClosure() {
     /**
      * @private
      */
+    _touchSwipeState: null,
+
+    /**
+     * @private
+     */
+    _touchSwipe: function PDFPresentationMode_touchSwipe(evt) {
```

Comment:
To prevent possible issues, I think that you want the following code at the top of this method:
```javascript
if (!this.active) {
  return;
}
```
to avoid handling a touch gesture until the browser fullscreen mode has actually been invoked.